graph/src/lib/clientGraph.ts

```diff
@@ -26,11 +27,17 @@ const model = new ChatOpenAI({
   openAIApiKey: env.VITE_OPENAI_API_KEY,
 });
 
+const StateAnnotation = Annotation.Root({
+  messages: Annotation<BaseMessage[]>({
+    reducer: messagesStateReducer,
+  }),
+});
```

Comment:
Brings in state annotation - for now only with messages, meaning this will be effectively the same as before, but the goal is to be able to reduce state with more fields here 🙏🏽